backend/http/http.go

```diff
@@ -538,14 +599,18 @@ func (o *Object) url() string {
 
 // stat updates the info field in the Object
 func (o *Object) stat(ctx context.Context) error {
-	if o.fs.opt.NoHead {
+	if o.fs.opt.NoHead && o.fs.opt.GoIndexDriveIndex < 0 {
```

Comment:
If this stays in the http backend, perhaps "NoHead" should be changed, because one would disable individual-file / folder POST requests instead.